tests/e2e/Services/VCS/VCSBase.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Tests\E2E\Services\VCS;
+
+trait VCSBase
```

Comment:
We define the trait but never use it.

I think you need `trait VCSBase extends Scope` here , and then in `tests/e2e/Services/VCS/VCSConsoleClientTest.php` you do `class VCSConsoleClientTest extends VCSBase`.

Reply:
I don't think `trait` supports `extends` syntax. Also, here the format I've followed is what we have in other tests like functions.